packages/xstate-react/src/useActor.ts

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from 'react';
+import { ActorRefLike, ActorRef, Sender } from './types';
+import { EventObject } from 'xstate';
+
+const defaultGetCurrentValue = <TEvent extends EventObject, TEmitted = any>(
+  actorLike: ActorRef<TEvent, TEmitted> | ActorRefLike<TEvent, TEmitted>
+): TEmitted => {
+  return 'current' in actorLike ? actorLike.current : (undefined as any); // V4 Actors do not have 'current'
+};
+
+export function useActor<TEvent extends EventObject, TEmitted = any>(
+  actorLike: ActorRef<TEvent, TEmitted> | ActorRefLike<TEvent, TEmitted>,
+  getCurrentValue: <TEvent extends EventObject, TEmitted = any>(
```

Comment:
should this be a public parameter? we expect a particular interface of an actor to be satisfied on the position of `actorLike` so it seems that we should be able to just use this information without having to accept `getCurrentValue` from a consumer